setup.cfg

```diff
@@ -44,7 +46,6 @@ docs =
 universal=0
 
 [tool:pytest]
-ignore=tests/fixtures/
```

Comment:
This line caused this warning...

```
.tox/py38/lib/python3.8/site-packages/_pytest/config/__init__.py:1233
  /home/jugmac00/Projects/venusian/.tox/py38/lib/python3.8/site-packages/_pytest/config/__init__.py:1233: PytestConfigWarning: Unknown config option: ignore
  
    self._warn_or_fail_if_strict(f"Unknown config option: {key}\n")

-- Docs: https://docs.pytest.org/en/stable/warnings.html
```

Removing it did not cause any noticable side effect.

Reply:
Hmmm, we should definitely make sure to ignore the fixtures folder though from being scanned for test cases, but this may be a hold-over from when this was nose tests?

Reply:
I had a look in pytest's documentation and also asked a core maintainer - there is no straightforward way to add a single directory to be excluded via configuration file.

As noted, `-ignore` is only allowed on the cli, and for the configuration there is https://docs.pytest.org/en/6.2.x/reference.html#confval-norecursedirs but when you set a directory, you overwrite all default ones.

So, either keep it as it is, add `-ignore` to the pytest commands, or define the defaults for `norecursedirs` ourselves.

Reply:
I just learned another way. We could add a `addopts` to the configuration file and add the cli option there.